tools/tfuser/main.go

```diff
@@ -104,34 +104,62 @@ func main() {
 						},
 						{
 							Name:  "add-node",
-							Usage: "add a node to a existing network",
+							Usage: "Add a node to a existing network",
 							Flags: []cli.Flag{
 								cli.StringFlag{
 									Name:  "node",
-									Usage: "node ID of the node to add to the network",
+									Usage: "Node ID of the node to add to the network",
 								},
 								cli.StringFlag{
 									Name:  "subnet",
-									Usage: "subnet to use on this node. The subnet needs to be included in the IP range of the network",
+									Usage: "Subnet to use on this node. The subnet needs to be included in the IP range of the network",
 								},
 								cli.UintFlag{
 									Name:  "port",
-									Usage: "Wireguar port to use. if not specified, tfuser will automatically check BCDB for free fort to use",
+									Usage: "Wireguard port to use. if not specified, tfuser will automatically check BCDB for free fort to use",
+								},
+								cli.BoolFlag{
+									Name:  "force-hidden",
+									Usage: "Forcibly mark the node as hidden, even if it is registered with public endpoints",
 								},
 							},
 							Action: cmdsAddNode,
 						},
 						{
 							Name:  "remove-node",
-							Usage: "prints the wg-quick configuration file for a certain user in the network",
+							Usage: "Removes a Network Resource from the network",
 							Flags: []cli.Flag{
 								cli.StringFlag{
 									Name:  "node",
-									Usage: "node ID to remove from the network",
+									Usage: "Node ID to remove from the network",
 								},
 							},
 							Action: cmdsRemoveNode,
 						},
+						{
+							Name:   "graph",
+							Usage:  "create a dot graph of the network",
+							Action: cmdGraphNetwork,
+						},
+						{
+							Name:  "add-access",
+							Usage: "Add external access to the network",
+							Flags: []cli.Flag{
```

Comment:
an extra flag should be added so the use can specify it's own wireguard public key instead of always generating a new one.